angr/analyses/cfg/cfg_node.py

```diff
@@ -1,12 +1,13 @@
 import traceback
 import logging
 
+from archinfo.arch_soot import SootAddressDescriptor
 import archinfo
 
 from ...codenode import BlockNode, HookNode
 from ...engines.successors import SimSuccessors
 
-_l = logging.getLogger(__name__)
+l = logging.getLogger(__name__)
```

Comment:
Is the logger used anywhere in this file?

Reply:
It is used in line `93`. I replaced it with `_l` as it was before.